chunky/src/java/se/llbit/chunky/ui/render/tabs/PostprocessingTab.java

```diff
@@ -122,9 +152,81 @@ public PostProcessingFilter fromString(String string) {
     exposure.clampMin();
     exposure.onValueChange(value -> {
       scene.setExposure(value);
-      scene.postProcessFrame(new TaskTracker(ProgressListener.NONE));
-      controller.getCanvas().forceRepaint();
+      applyChangedSettings(false);
+    });
+    hableCurveSettings.managedProperty().bind(hableCurveSettings.visibleProperty());
+    gdcPreset.setOnAction((e) -> {
+      if (scene.postProcessingFilter instanceof HableToneMappingFilter) {
+        ((HableToneMappingFilter) scene.postProcessingFilter).applyPreset(HableToneMappingFilter.Preset.GDC);
+        applyChangedSettings(true);
+      }
+    });
+    fwPreset.setOnAction((e) -> {
+      if (scene.postProcessingFilter instanceof HableToneMappingFilter) {
+        ((HableToneMappingFilter) scene.postProcessingFilter).applyPreset(HableToneMappingFilter.Preset.FILMIC_WORLDS);
+        applyChangedSettings(true);
+      }
+    });
+    ue4CurveSettings.managedProperty().bind(ue4CurveSettings.visibleProperty());
+    acesPreset.setOnAction((e) -> {
+      if (scene.postProcessingFilter instanceof UE4ToneMappingFilter) {
+        ((UE4ToneMappingFilter) scene.postProcessingFilter).applyPreset(UE4ToneMappingFilter.Preset.ACES);
+        applyChangedSettings(true);
+      }
     });
+    ue4LegacyPreset.setOnAction((e) -> {
+      if (scene.postProcessingFilter instanceof UE4ToneMappingFilter) {
+        ((UE4ToneMappingFilter) scene.postProcessingFilter).applyPreset(UE4ToneMappingFilter.Preset.LEGACY_UE4);
+        applyChangedSettings(true);
+      }
+    });
+
+    EventHandler<KeyEvent> postprocessingSettingsHandler = e -> {
+      if (e.getCode() == KeyCode.ENTER) {
+        if (scene.postProcessingFilter instanceof HableToneMappingFilter) {
+          HableToneMappingFilter filter = (HableToneMappingFilter) scene.postProcessingFilter;
+          filter.setShoulderStrength((float) hableShoulderStrength.valueProperty().get());
```

Comment:
I believe it is possible to do `.valueProperty().floatValue()`. If so, would that be better?